code/game/objects/obj_defense.dm

```diff
@@ -13,6 +39,95 @@
 		if(BURN)
 			playsound(src.loc, 'sound/items/welder.ogg', 100, 1)
 
+/obj/hitby(atom/movable/AM)
+	..()
+	var/tforce = 0
+	if(ismob(AM))
+		tforce = 10
+	else if(isobj(AM))
```

Comment:
make this an explicit `istype(AM, /obj)`; `isobj` is actually a BYOND proc, and it can include some things it shouldn't (unless you want to override the isobj check like TG did...but I'm not sure the ramifications of that.